gtsfm/scene_optimizer.py

```diff
@@ -367,6 +382,12 @@ def aggregate_frontend_metrics(
         num_entries,
     )
 
+    logger.debug(
+        "[Two view optimizer] [Summary] All correct correspondences: %d/%d",
```

Comment:
Maybe a clearer log message is: `Number of image pairs with 100% inlier ratio:`

Reply:
oh yeah. this is way better :P